src/Acl.php

```diff
@@ -30,12 +30,6 @@ public function getRules(Model $model)
         /** @var \Atk4\Login\Model\User */
         $user = $this->auth->user;
 
-        if (!$user->isLoaded()) {
```

Comment:
what do you expect from l38:

```
$res = $user->ref('AccessRules')->addCondition('model', 'in', $modelClasses);
```

returned then?

Reply:
here is a comparison of query issues by the l38 for logged & un-logged user:

![image](https://user-images.githubusercontent.com/2228672/155708272-ed40ea30-cc94-47f8-bd8f-bb93f25cf0ea.png)

Reply:
related with https://github.com/atk4/data/pull/969